Freshen up

JustinAiken opened the pull request:
- [x] Fill in README with usage instructions 
- [x] Set up CI
  - [x] TravisCI for test suite
  - [x] Coveralls
  - [x] Code Climate
- [X] Relax Rake restriction
- [x] ⬆️ Version bump!

JustinAiken committed:
[DOC] Fill in README a bit

JustinAiken committed:
Ignore downloaded file

JustinAiken committed:
Add travis config file

JustinAiken committed:
Add coveralls...

JustinAiken committed:
Make Travis actually run the tests

JustinAiken committed:
Remove pre-2.3 rubies from testing matrix...

- Given a choice between supporting EOL rubies, and giving up squiggly
heredocs... goodbye to the past!

JustinAiken committed:
[DOC] Add badges for Travis/CC/Coveralls

JustinAiken committed:
Relax rake dependency..

- Just about any version of rake that can be installed with the
supported rubies will do what we need

JustinAiken committed:
Unblock the path to rubygems

JustinAiken committed:
[DOC] Make badges clickable

JustinAiken committed:
UserTesting

JustinAiken committed:
Version bump :arrow_up: 0.1.2

JustinAiken committed:
Make the logo show up

JustinAiken committed:
[DOC] Add a changelog

cdimartino commented on README.md:
```diff
@@ -1,35 +1,105 @@
 
-TODO: Delete this and the text above, and describe your gem
+
+This gem is a Ruby wrapper around `@dcoker`'s [biscuit library](https://github.com/dcoker/biscuit), a multi-region HA key-value store for your AWS infrastructure secrets.
```
`s/dcoker/docker/`

JustinAiken replied:
Nope, it's not by docker, it's by a Mr. Coker

cdimartino replied:
🤦‍♂ my bad :shipit:

cdimartino commented on README.md:
```diff
@@ -1,35 +1,105 @@
 
+## License
+
+MIT.
```
Should this be a [link to the MIT license](https://opensource.org/licenses/MIT)?

cdimartino commented on README.md:
```diff
@@ -1,35 +1,105 @@
+
+MIT.
+
+Library created by [UserTesting](https://usertesting.com)
```
This feels like an awesome place to link to the [UserTesting Developer Pages](https://developers.usertesting.com) but we'll need to create them 🤷‍♂

JustinAiken replied:
![developers](https://user-images.githubusercontent.com/1568662/65074600-69f94400-d952-11e9-8648-3256c69f5304.gif)

cdimartino reviewed (approved):
lgtm

JustinAiken committed:
[CR] :link: the license

perryism reviewed (approved):
Looks good!

JustinAiken committed:
[BUGFIX] Handle values that contain : in them...

- When splitting up the pairs, limit it to 2 pieces.  That way the pairs
are really pairs

nevinera reviewed (approved):
nice and simple

dakotahp commented on biscuit.gemspec:
```diff
@@ -30,5 +22,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
 
-  spec.add_runtime_dependency "rake", "~> 10.0"
+  spec.add_runtime_dependency "rake"
```
👏🏼 I was just researching upgrading our rake gem in the orders repo and this was a major blocker. Big fan of this change.

dakotahp reviewed (approved):
Nice improvements
